xbmc/video/VideoDatabase.cpp

```diff
@@ -8155,6 +8155,7 @@ unsigned int CVideoDatabase::GetMusicVideoIDs(const std::string& strWhere, std::
     std::string strSQL = "select distinct idMVideo from musicvideo_view";
     if (!strWhere.empty())
       strSQL += " where " + strWhere;
+    strSQL += PrepareSQL(" ORDER BY RANDOM()");
```

Comment:
Why PrepareSQL?

Reply:
To allow for the syntax differences between SQLite and MySQL.
SQLite has "ORDER BY RANDOM()"
MySQL has "ORDER BY RAND()"

Reply:
I half-assumed something like that... Not particularly nice, but not likely to change now either :-/
Can you add a comment at least?